app/components/finance/npq/payment_overviews/payment_overview_summary.rb

```diff
@@ -77,13 +77,13 @@ def output_payments
         def statement_declarations
           if statement.current?
             ParticipantDeclaration::NPQ
-              .eligible_for_lead_provider(npq_lead_provider)
+              .for_lead_provider(npq_lead_provider)
+              .eligible
               .where(statement_id: nil)
           else
             statement
               .participant_declarations
               .paid_payable_or_eligible
-              .unique_id
```

Comment:
you may want to keep an eye on this. i fear this may change the value of paid out statements